src/compas_view2/objects/polylineobject.py

```diff
@@ -7,10 +7,11 @@ class PolylineObject(BufferObject):
     default_color_points = [0.1, 0.1, 0.1]
     default_color_line = [0.4, 0.4, 0.4]
 
-    def __init__(self, data, pointcolor=None, linecolor=None, color=None, **kwargs):
+    def __init__(self, data, closed=False, pointcolor=None, linecolor=None, color=None, **kwargs):
```

Comment:
what is the meaning of "closed"? from the snippets below it is not clear to me.

Reply:
this is to whether close the polyline by connecting start to end

Reply:
should it then not be "close" rather than "closed"?